commands/command.go

```diff
@@ -48,14 +48,22 @@ func (c *Command) ChildCommands() []*Command {
 
 // CmdBuilder builds a new command.
 func CmdBuilder(parent *Command, cr CmdRunner, cliText, desc string, out io.Writer, options ...cmdOption) *Command {
-	return cmdBuilderWithInit(parent, cr, cliText, desc, out, true, options...)
+	return cmdBuilderWithInit(parent, cr, cliText, desc, desc, out, true, options...)
 }
 
-func cmdBuilderWithInit(parent *Command, cr CmdRunner, cliText, desc string, out io.Writer, initCmd bool, options ...cmdOption) *Command {
+// CmdBuilderWithDocs builds a new command with custom long descriptions.
+// This was introduced to transition away from the current CmdBuilder
+// implementation incrementally. When all commands are built using this
+// function, CmdBuilderWithDocs should replace it.
+func CmdBuilderWithDocs(parent *Command, cr CmdRunner, cliText, shortdesc string, longdesc string, out io.Writer, options ...cmdOption) *Command {
```

Comment:
Was going to comment that I know the _builder_ pattern has ended up being a real pain as we try to refactor this codebase and get rid of it _but_ I think you took the best approach you could given the constraints of what we have today.

Docs are worlds and away better than what we have today. I don't really see introducing this additional function as making it any harder for us to get rid of this builder pattern and we get the benefit of having nice doc strings.

Reply:
That's the hope, @zachgersh! <3